src/components/Lightbox/Lightbox.tsx

```diff
@@ -35,7 +35,7 @@ export const Lightbox = ({ baseUrl, imageData }: LightboxProps) => {
         sizes="100vw"
         // sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
         // fill
-        alt={imageData.caption ?? 'article-image'}
+        alt={imageData.alternativeText ?? 'article-image'}
```

Comment:
https://github.com/jumperexchange/jumper-exchange/pull/1659/files#diff-7575f8afc1988d09c9b2e6b2509cf99e44ab2607fc19026543a36c89915f972eR31 says that imageData can be undefined but not there